### PR TITLE
Make responsive images optional and provide a way to turn it off

### LINKED ISSUE #52: New utility to turn off responsive images
Provide a way to remove this via a utility: https://github.com/westfieldlabs/scally/blob/master/core/base/_media.scss#L34.

### PR DESCRIPTION
**Pull request by chris-pearce**
This closes #52.

### CODE REVIEW

File: core/base/_media.scss [hunk 1]
```diff
@@ -31,20 +38,33 @@ video {vertical-align: middle;}
...
+
+	font-style: italic; // [2]
 }
 
 
 /**
+ * If responsive images are turned on then use this to turn it off.
+ */
+
+.img-none-responsive {max-width: none;}
```
Comment (AlecRaeside) [inline core/base/_media.scss]: I find this class name confusing; something like `.img-not-responsive` makes more sense when reading the class, unless I'm missing something?

Comment (chris-pearce) [inline core/base/_media.scss]: Yeah `not` is more readable.

**Comment by AlecRaeside** [comment]
LGTM

**Comment by kllevin** [comment]
LGTM

**Pull request merged**